### PR TITLE
Fix conda/pip qt5 conflict

### PR DESCRIPTION
**Pull request by JoOkuma**

### CODE REVIEW

**Comment by JoOkuma** [comment]
Fixes #169

File: pyproject.toml [hunk 1]
```diff
@@ -46,7 +46,6 @@ dependencies = [
     "websocket >= 0.2.1",
     "websockets >= 12.0",
     "zarr >=2.15.0,<3.0.0",
-    "pyqt5 >=5.15.4",
```
Comment (psobolewskiPhD) [inline pyproject.toml:49]: I think you should add `qtpy` because you import from it.

Comment (psobolewskiPhD) [reply pyproject.toml:49]: It's a transitive dependency right now, so everything works.

Comment (JoOkuma) [reply pyproject.toml:49]: thanks, @psobolewskiPhD, you did mention the issue, and I forgot to include it
I'm fixing the errors in a different PR and this should be good soon.

Happy holidays!

**Pull request merged**